torcharrow/functional.py

```diff
@@ -141,6 +141,60 @@ def add_tokens(
     return _dispatch("add_tokens", input_col, tokens, begin)
 
 
+def bpe_tokenize(
+    tokenizer,
+    input_col: Union[ListColumn, List[str]],
+) -> ListColumn:
+    """
+    Tokenize text into a list of tokens using the GPT-2 BPE Tokenizer
```

Comment:
I think we need to make clearly in docstring that this method is only useable if TA is built with `USE_TORCH=1`?